packages/util/src/environment.ts

```diff
@@ -138,6 +138,7 @@ export function isNodeSdk(): boolean {
 export function isSafari(): boolean {
   return (
     !isNode() &&
+    navigator.userAgent !== undefined && // not react-native
```

Comment:
Please just remove this comment and then LGTM. Thanks for the contribution and my apologies for breaking you!